### ISSUE TITLE
Can't install reservoir due to drupal-core-strict

### ISSUE THREAD

**Issue by idiazroncero** [acquia/reservoir#63]
Hi;

I can't see the way to install reservoir following the official docs. There seems to be an issue related to #56 . 

With composer updated on its stable channel (1.5.2 as today), there is a dependency problem which prevents installation:

```
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - webflo/drupal-core-strict 8.4.0 requires composer/installers v1.4.0 -> satisfiable by composer/installers[v1.4.0].
    - webflo/drupal-core-strict 8.4.0-rc2 requires composer/installers v1.4.0 -> satisfiable by composer/installers[v1.4.0].
    - webflo/drupal-core-strict 8.4.x-dev requires composer/installers v1.4.0 -> satisfiable by composer/installers[v1.4.0].
    - webflo/drupal-core-strict 8.5.x-dev requires composer/installers v1.4.0 -> satisfiable by composer/installers[v1.4.0].
    - don't install composer/installers v1.4.0|don't install composer/installers v1.2.0
    - acquia/reservoir 1.0.0-alpha2 requires composer/installers 1.2.0 -> satisfiable by composer/installers[v1.2.0].
    - Conclusion: don't install webflo/drupal-core-strict 8.3.7
    - Installation request for acquia/reservoir 1.0.0-alpha2 -> satisfiable by acquia/reservoir[1.0.0-alpha2].
    - webflo/drupal-core-strict 8.3.4 requires asm89/stack-cors 1.0.0 -> satisfiable by asm89/stack-cors[1.0.0].
    - webflo/drupal-core-strict 8.3.5 requires asm89/stack-cors 1.0.0 -> satisfiable by asm89/stack-cors[1.0.0].
    - webflo/drupal-core-strict 8.3.6 requires asm89/stack-cors 1.0.0 -> satisfiable by asm89/stack-cors[1.0.0].
    - webflo/drupal-core-strict 8.3.x-dev requires asm89/stack-cors 1.0.0 -> satisfiable by asm89/stack-cors[1.0.0].
    - Conclusion: don't install asm89/stack-cors 1.0.0|install webflo/drupal-core-strict 8.3.7
    - acquia/reservoir 1.0.0-alpha2 requires webflo/drupal-core-strict ^8.3.4 -> satisfiable by webflo/drupal-core-strict[8.3.4, 8.3.5, 8.3.6, 8.3.7, 8.3.x-dev, 8.4.0, 8.4.0-alpha1, 8.4.0-beta1, 8.4.0-rc1, 8.4.0-rc2, 8.4.x-dev, 8.5.x-dev].
    - webflo/drupal-core-strict 8.4.0-alpha1 requires doctrine/annotations v1.2.7 -> satisfiable by doctrine/annotations[v1.2.7].
    - webflo/drupal-core-strict 8.4.0-beta1 requires doctrine/annotations v1.2.7 -> satisfiable by doctrine/annotations[v1.2.7].
    - webflo/drupal-core-strict 8.4.0-rc1 requires doctrine/annotations v1.2.7 -> satisfiable by doctrine/annotations[v1.2.7].
    - Conclusion: don't install doctrine/annotations v1.2.7|install webflo/drupal-core-strict 8.3.4|install webflo/drupal-core-strict 8.3.5|install webflo/drupal-core-strict 8.3.6|install webflo/drupal-core-strict 8.3.7|install webflo/drupal-core-strict 8.3.x-dev
```

The official drupal-core-strict repo doesn't helps too much:

```
To fix to dependencies tested on Drupal 8.3.0 add "webflo/drupal-core-strict": "8.3.0" to the require section of your composer.json and run composer update from the command line.

In other words: this is a virtual package, that causes you to get exactly the versions of Drupal core's dependencies as they are specified in Drupal core's composer.lock file.

Using composer require is problematic on an existing site.
```

Any info?

**Comment by ghost** [comment]
Hi @idiazroncero, 

I solved temporarily this issue by adding `webflo/drupal-core-strict": "8.4.0"` and changing `composer/installers": "^1.0"` to `composer/installers": "~1.2"` into my composer.json

Best regards.
